config.schema.json

```diff
@@ -218,11 +219,40 @@
       "$ref": "#/$defs/additional_ibc_token_info"
     }
   },
+  "demex_trading_league_config": {
+    "type": "object",
+    "description": "Config for Demex Trading League",
+    "properties": {
+      "type": "object",
+      "promoMarkets": {
+        "type": "array",
+        "description": "List of market names designated as Promo markets",
+        "items": {
+          "$ref": "#/$defs/promo_market"
+        }
+      },
+      "currentMarketSymbol": {
+        "type": "string",
+        "description": "Current market symbol used for main trading league market icon"
```

Comment:
Rewrite this description based on the revised description on L74 of `.github/markets/pr_template.md`